pkgs/misc/emulators/ppsspp/default.nix

```diff
@@ -26,6 +27,14 @@ mkDerivation rec {
     sha256 = "sha256-vfp/vacIItlPP5dR7jzDT7oOUNFnjvvdR46yi79EJKU=";
   };
 
+  patches = [
+    # adds ffmpeg 4.3 compatibility, remove for ppsspp 1.12
+    (fetchpatch {
+      url = "https://patch-diff.githubusercontent.com/raw/hrydgard/ppsspp/pull/14176.patch";
```

Comment:
I **think** you can get the same effect by specifying the merge-commit of the PR: https://github.com/hrydgard/ppsspp/commit/8a69c3d1226fe174c49437514a2d3ca7e411c3fa.patch, and that way we have a more permanent identifier (in theory someone could delete the whole github project one day)

Reply:
Thanks, tested & updated it!